scripting/lua/api/ServerCb.cpp

```diff
@@ -26,22 +26,22 @@ const std::vector<ServerCbProxy::CustomRegType> ServerCbProxy::REGISTER_CUSTOM =
 {
 	{
 		"addToBattleLog",
-		&ServerCbProxy::apply<BattleLogMessage>,
+		(lua_CFunction)(&ServerCbProxy::apply<BattleLogMessage>),
```

Comment:
Otherwise, it causes strange errors: "Can not cast overloaded_function to lua_CFunction"